client/src/pages/RoomPage/index.tsx

```diff
@@ -210,7 +210,7 @@ function RoomPage({ name }: { name: string }): JSX.Element {
       stream.getAudioTracks()[0].enabled = !selfUserInfoRef.current.mute;
     }
     networkRef.current?.updateInfo(selfUserInfoRef.current);
-  }, [selfUserInfoRef.current.mute, stream]);
+  }, [selfUserInfoRef.current, stream]);
```

Comment:
The stream is unneeded too since then, the avatar position won't be updated on mute.